### DISCUSSION TITLE
Move the robotic arm to open the cabinet

### DISCUSSION THREAD

**Discussion by duidui6666** [isaac-sim/IsaacLab#350] [Q&A]
Is the task of opening a cabinet with a mobile robotic arm updated to github in the paper video? I want to try the same task with my own robot model

**Comment by Bailey-24** [comment]
how to add your own robot to Isaac lab.

**Reply by WangYCheng23** [reply]
you should write your own robot config

**Reply by Mayankm96** [reply]
Please check the how-to section in the documentation: https://isaac-sim.github.io/IsaacLab/source/how-to/index.html
